srv/components/misc_pkgs/kafka/start.sls

```diff
@@ -31,9 +30,8 @@ Wait for zookeeper to start:
       - Start zoopkeper
 
 Start kafka:
-  cmd.run:
-    - name: ./bin/kafka-server-start.sh -daemon config/server.properties
-    - cwd: /opt/kafka
-    - unless: test 1 -le $(ps ax | grep ' kafka\.Kafka ' | grep java | grep -v grep | awk '{print $1}' | wc -l)
+  service.running:
```

Comment:
Don't delete
Maintain the commands commented, just in case we have to fallback.